bolt-modules/boltlib/lib/puppet/functions/set_var.rb

```diff
@@ -0,0 +1,30 @@
+# Sets a variable { key => value } for a target
+#
+# This function takes 3 parameters:
+# * A Target object to set the variable for
+# * The key for the variable (String)
+# * The value of the variable (Data)
+#
+# Returns nil
```

Comment:
The docs comment needs to be adjacent to the call to create_function, now there is a `require` between it and the call.

Reply:
This is true for most of the functions in this repo. Should I change them all in this PR? Does that break yarddocs?

Reply:
That would be nice to fix as an additional maint commit.